vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,98 @@
+
+### `status` file
+
+The vcpkg database, this file contains an entry for every package installed for every triplet. This
+file is automatically updated by vcpkg and should not be modified manually.
+
+## Triplet directories
+
+The output of each package installation is contained in a triplet-specific directory, for example,
+packages installed for the `x64-windows` triplet are located in `installed/x64-windows` directory.
+
+The layout for the subdirectories inside each triplet directory is the same:
+
+| Subdirectory            | File type                                         | 
+| ----------------------- | ------------------------------------------------- |
+| `bin`                   | Release `.dll` and `.pdb` files                   |
+| `debug/bin`             | Debug `.dll` and `.pdb` files                     |
+| `debug/lib`             | Debug `.lib`, `.so`, and `.a` files               |
+| `debug/lib/manual-link` | Manually-linkable debug `.dll` and `.pdb` files   |
+| `debug/lib/pkgconfig`   | pkgconfig files (`.pc`)                           |
+| `include`               | Header-files (`.h`, `.hpp`, `.hxx`)               |
+| `lib`                   | Release `.lib`, `.so`, and `.a` files             |
+| `lib/manual-link`       | Manually-linkable release `.dll` and `.pdb` files |
+| `lib/pkgconfig`         | Pkgconfig files (`.pc`)                           |
+| `share/<port>`          | Miscellaneous files.                              |
+| `share/pkconfig`        | pkgconfig files (`.pc`)                           |
+| `tools/<port>`          | Executable tools                                  |
+
+
+### `bin` and `debug/bin` directories
+
+On Windows, these directories contain DLL and PDB files for release and debug configuration
+respectively.
+
+### `lib` and `debug/lib` directories
+
+Contain object files, any file with the `.lib`, `.so`, and `.a` extensions is placed in these
+configuration-dependent directories.
+
+### `lib/manual-link` and `debug/lib/manual-link` directories
+
+Contain object files that must be manually linked, these directories are configuration dependent.
+Each port must make sure to place manually-linkable files in these directories explicitly.
+
+### `lib/pkgconfig` and `share/pkgconfig` directories
+
+Contain pkconfig integration files (`.pc`).
+
+### `share/<port>`
+
+Contains miscellaneous files installed by each port, at the very least, each port is required to
+provide a `LICENSE` file in `share/<port>` (replace `<port>` with the port name). Some examples of
+files contained in this directory are: CMake integration files,
+[usage](../maintainers/handling-usage-files.md) files, SPDX files, etc.
+
+### `tools/<port>`
+
+Contains executable tools produced by a port, it is not required, but is highly recommended that
+each installed executable goes into a subdirectory matching the name of the port that produced it.
+For example, the `minizip` ports installs `minizip.exe` to
```

Comment:
I think it would be better to be less concrete here so we can demonstrate better:
```md
For example, a `contoso` port might install `ContosoGenerator.exe` to `tools/contoso/ContosoGenerator.exe`.
```